test/integration/viewer_spec.mjs

```diff
@@ -454,4 +454,346 @@ describe("PDF viewer", () => {
       );
     });
   });
+
+  describe("Detail view on zoom", () => {
+    function setupPages(zoom, maxCanvasPixels, width, height) {
+      const { resolve, reject, promise } = Promise.withResolvers();
+
+      let pages;
+
+      beforeAll(async () => {
+        const pagesP = loadAndWait(
+          "colors.pdf",
+          null,
+          zoom,
+          {
+            // TODO: Why is this needed?
+            earlySetup: () => {
+              window.devicePixelRatio = 2;
+            },
+          },
+          { maxCanvasPixels },
+          { height, width, devicePixelRatio: 2 }
+        );
+        pagesP.then(resolve, reject);
+        pages = await pagesP;
+      });
+
+      afterAll(async () => {
+        await closePages(pages);
+      });
+
+      return promise;
+    }
+
+    function extractCanvases(pageNumber) {
+      const toHex = ([r, g, b]) =>
+        `#` +
+        r.toString(16).padStart(2, "0") +
+        g.toString(16).padStart(2, "0") +
+        b.toString(16).padStart(2, "0");
```

Comment:
I suppose that it's not easily possible to use our existing helper function for this?
https://github.com/mozilla/pdf.js/blob/016de7422924e3f62d07c826965e549fcea0ff03/src/shared/util.js#L673-L675

Reply:
No, also because this code is passed to `page.evaluate` so I cannot simply import `Util` in the file. It would require adding a `PDFJSDev.test("TESTING")` check where that `Util` class is defined, to expose it as on `globalThis.Util`.

Reply:
It *should* already be available through `globalThis.pdfjsLib.Util`, but maybe that still isn't helpful here?

Reply:
That helped :)